src/components/single_use_components/InvestCarousel.js

```diff
@@ -119,7 +132,15 @@ export default function InvestCarousel() {
           </TouchableOpacity>
         ))}
       </View>
-      <InvestItemList listData={listData} isLoading={isLoading} />
+      {selectedCarouselOption === "CAROUSEL_TAB_INVESTORS" ? (
```

Comment:
I think allowing the user to switch between Investors, Jobs, and History tabs by swiping left or right would improve usability

Reply:
Thats a good point, was thinking about that. Gonna see how that can be impleneted